packages/site/src/providers/AtlantisPreviewEditorProvider.tsx

```diff
@@ -360,23 +360,24 @@ const skeletonHTML = (theme: Theme, type: "web" | "mobile") => {
 <head>
 <style>
 html,body,#root {
-  height: 100%;
-  width: 99%;
+  height: 100vh;
+  width: 100vw;
   display: flex;
   align-items: center;
   justify-content: center;
   flex-direction: column;
-  min-height: 200px;
+  min-height: 280px;
```

Comment:
Now that the padding has gone from 48 down to 8, we have 80px more of vertical space to work with for free